src/examples/color-picker/ColorpickerDialogDefault.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { ColorpickerDialog } from '@zendeskgarden/react-colorpickers';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const Example = () => (
+  <Row>
+    <Col textAlign="center">
+      <ColorpickerDialog
+        aria-label="Color picker dialog button"
```

Comment:
[nit] maybe something closer to reality, like "Select your favorite color"

But more important, did you test this? The props are spread on to the `Colorpicker` div – as expected. But this might identify a short-coming in the API. Perhaps the need for `triggerProps` (or something better?) on the underlying component. Otherwise, VO reads "button".

Reply:
I hadn't noticed that props are not passed through to the trigger. There is a new PR (https://github.com/zendeskgarden/react-components/pull/1071) in draft mode that allows consumers to apply props to the trigger.

Reply:
The `aria-label` has been removed from this PR. I'll open a new PR to add the `aria-label` via `buttonProps` once  https://github.com/zendeskgarden/react-components/pull/1071 is released.